### ISSUE TITLE
Pluralization rule for {one} is not applied for -1 value

### ISSUE THREAD

**Issue by pkuczynski** [formatjs/react-intl#1213]
### Bug

Pluralization for `{one}` is not applied for negative 1 (-1) although semantically its the same thing.

### Expected behavior

 For example for the following plural EN message

`{value, plural, zero {boxes} one {box} other {boxes}}`

I would expect to receive (depending on the `value` passed):

* **-1 box**
* 0 boxes
* 1 box
* 5 boxes

### Current behavior

* **-1 boxes**
* 0 boxes
* 1 box
* 5 boxes

### Your Environment

* macOS Mojave
* Chrome 70
* react-intl@2.7.2
* react@16.6.0

**Comment by papasmile** [comment]
HI, @pkuczynski, how about `{value, plural, =-1 {box} zero {boxes} one {box} other {boxes}}`?

As far as making that default behavior: that rule makes sense for English but might not be universal linguistically.  Would be interesting to find out if there were special language rules for negative numbers...

**Comment by pkuczynski** [comment]
This would make sense for -1, but how about -2, -3, and -113... Can't make `=` rule for everything, can I? :) I believe pluralization should work for both positive and negative values the same way...

**Comment by papasmile** [comment]
OK, sure; as far as I understand CLDR does not support negative numbers in most of it's functions.  So I suppose you would have to do something like:

```
const formatWithNegative number => (
  (number < 0 ? '-' : '') + intl.formatNumber(Math.abs(number), MY_PATTERN)
);
...
render() {
  <span>{{ formatWithNegative(myNum) }}</span>
}
```

As far as what ** is ** available in CLDR, there are some general notes on some custom formats for negative number support at: http://cldr.unicode.org/translation/number-patterns

**Comment by longlho** [comment]
fixed in #1301. The spec only considers absolute value. Native browser should already work and we've also fixed the polyfill as well https://github.com/eemeli/intl-pluralrules/pull/6
